types/__tests__/type-tests.ts

```diff
@@ -178,15 +178,9 @@ export async function testByRole() {
   console.assert(queryByRole(element, 'button', {name: /^Log/}) === null)
   console.assert(
     queryByRole(element, 'button', {
-      name: (name, el) => name === 'Login' && el.hasAttribute('disabled'),
+      name: (name, el) => name === 'Login' && !!el?.hasAttribute('disabled'),
     }) === null,
   )
-
-  // allow to query for a role that isn't included in the types
-  console.assert(queryByRole(element, 'foo') === null)
-  console.assert(queryByRole(element, /foo/) === null)
-  console.assert(screen.queryByRole('foo') === null)
-  console.assert(screen.queryByRole(/foo/) === null)
```

Comment:
Why remove these?

Reply:
Why is this not allowed? `query*` can return a nullable value

Reply:
Because "foo" is not a valid role according to types